app/api/files/jsRoutes.js

```diff
@@ -132,4 +132,79 @@ export default app => {
       res.json('ok');
     }
   );
+
+  const parseQueryProperty = (query, property) =>
+    query[property] ? JSON.parse(query[property]) : query[property];
+
+  const generateExportFileName = databaseName => `${databaseName}-${new Date().toISOString()}.csv`;
+
+  app.get(
+    '/api/export',
+
+    validation.validateRequest(
+      Joi.object().keys({
```

Comment:
@joan should we start using the new JSON schema here or keep JOI for old route files (even if the endpoint is new)?

Reply:
@RafaPolit @fnocetti i think we should remove gradually the joi validations, for sure for new routes use the new one, example here:

https://github.com/huridocs/uwazi/blob/988b5ddd02705852babfe1a5b925bb7c9ace3b7e/app/api/files/routes.ts#L98-L106